rest_framework_json_api/relations.py

```diff
@@ -224,6 +225,43 @@ def get_choices(self, cutoff=None):
         ])
 
 
+class PolymorphicResourceRelatedField(ResourceRelatedField):
```

Comment:
Since this is one of the primary APIs that is exposed to support polymorphic models, I think it should have a docstring describing its function/purpose and any other things that a potential user needs to know.